OracleDatabase/RAC/OracleConnectionManager/dockerfiles/21.3.0/configCMAN.sh

```diff
@@ -0,0 +1,273 @@
+#!/bin/bash
+# LICENSE UPL 1.0
+#
+# Copyright (c) 1982-2019 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Please change the copyright years to be accurate. 1982 is not the year this file was created and it was updated in 2022 and please remove "All rights reserved".